### PR TITLE
copy edit

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

File: draft-ietf-uta-rfc6125bis.md [hunk 1]
```diff
@@ -176,35 +175,10 @@ reference identity so it can deterministically and automatically authenticate th
 
 This document defines procedures for how clients do this verification.
 It therefore also defines requirements on other parties, such as
-the CA's that issue certificates, the service administrators requesting
+the certification authorities that issue certificates, the service administrators requesting
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:178]: Why not use CA here?  Just curious.

File: draft-ietf-uta-rfc6125bis.md [hunk 2]
```diff
@@ -244,24 +219,24 @@ The rules are brief:
...
 could support the TLS server making requests to the TLS client after the
 TLS handshake; these is no requirement that the roles at the application
-layer match the TLS-layer.
+layer match the TLS layer.
 
 At the time of this writing, other protocols such as {{QUIC}} and
 Network Time Security ({{NTS}}) use DTLS or TLS to do the
 initial establishment of cryptographic key material.
-Such services MUST also follow the rules specified here.
+The rules specified here are intended to apply to such services, as well.
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:239]: s/are intended to// ?

**Review by richsalz — COMMENTED** [review]
approved even without these suggestions.

File: draft-ietf-uta-rfc6125bis.md [hunk 3]
```diff
@@ -167,7 +166,7 @@ it has some notion of the server's
 identity (e.g., "the website at example.com") while attempting to establish
 secure communication.  Likewise, during TLS negotiation, the server presents
 its notion of the service's identity in the form of a public-key certificate
-that was issued by a CA in the context of the
+that was issued by a certificate authority (CA) in the context of the
```
[the remark this answers is not in the document]
Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md:169]: I don't care which we use, let's just be consistent. :-) It seems that "certificate authority" is more common.

[the remark this answers is not in the document]
Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md:169]: Fixed via https://github.com/richsalz/draft-ietf-uta-rfc6125bis/pull/40/commits/454874378e4ee384a2651e8e7a16bca23b6844bb

File: draft-ietf-uta-rfc6125bis.md (hunk 1 above)
Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md]: We use the term so infrequently that I didn't see much use for the abbreviation. :shrug:

**Comment by richsalz** [comment]
My bigger question is why spell it out after the first time; why not just use `CA` and `CAs` etc.

**Review by richsalz — APPROVED** [review]
Looks fine to me.

**Comment by stpeter** [comment]
> My bigger question is why spell it out after the first time; why not just use `CA` and `CAs` etc.

We need to expand the acronym on first use and then we use it only a few more times. Not a hill for me to die on either way. ;-)

**Comment by stpeter** [comment]
> The RPC will fix it up if needed ☺

Rightio. Shall we merge?

**Comment by richsalz** [comment]
merge away

**Pull request merged**